modules/imaging_browser/test/TestPlan.md

```diff
@@ -22,7 +22,8 @@
 
 ### C. Main panel:  per acquisition:
 
-16. Files can be downloaded (links clickable) only IFF has permission (future feature not implemented yet)
+16. Files can be downloaded (links clickable) only IFF has permission. Ensure that DICOM downloads are
```

Comment:
I agree with you but I think it would be good to keep this PR limited to the changes introduced in #4157.

I have already made a separate [ticket](https://github.com/aces/Loris/issues/4190) for someone to rewrite this test plan because the language isn't clear.